tests/core/notification/test_events_published.py

```diff
@@ -176,7 +185,11 @@ def test_events_published_for_scenario_submission():
     # 1 submission update event for jobs
     # 3 submission update events (for status: PENDING, RUNNING and COMPLETED)
     # 1 submission update event for is_completed
-    scenario.submit()
+    if standalone:
+        Orchestrator().run()
```

Comment:
why do we only start the `Orchestrator()` in standalone mode and not development mode?